src/core/services/loan-analytics.js

```diff
@@ -145,14 +159,16 @@ export default class LoanAnanlytics {
    * @memberof LoanAnanlytics
    */
   sendEvent(eventCategory, eventAction, label, extraParams) {
-    // log(
-    //   'eventCategory:-',
-    //   eventCategory,
-    //   '\t||\teventAction:-',
-    //   eventAction,
-    //   '\t||\tlabel:-',
-    //   label
-    // );
+    log(
+      'eventCategory:-',
+      eventCategory,
+      '||\teventAction:-',
+      eventAction,
+      '||\tlabel:-',
+      label,
+      '||\textraParams:-',
+      extraParams,
+    );
```

Comment:
QUESTION: Is this intended to remain live or should it be commented out?

Reply:
This is a console statement which only executes on local/dev URLs. on prod URL, it won't execute. So I needed to see some console execution to make sure new events being dispatching. So we don't have any issues by going these sinces these for local/dev URLs.